package.json

```diff
@@ -2,11 +2,37 @@
   "name": "trunkio-plugins",
   "version": "1.0.0",
   "scripts": {
-    "trunk": "trunk"
+    "trunk": "trunk",
+    "build-test": "tsc",
+    "test": "jest"
+  },
+  "engines": {
+    "node": ">=10.0.0"
   },
   "devDependencies": {
     "@commitlint/config-conventional": "^17.1.0",
-    "@trunkio/launcher": "^1.1.1"
+    "@trunkio/launcher": "^1.1.1",
+    "@types/jest": "^29.2.3",
+    "@types/node": "^18.11.10",
+    "@types/yaml": "^1.9.7",
+    "@typescript-eslint/eslint-plugin": "^5.45.1",
+    "eslint": "^8.29.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-standard-with-typescript": "^23.0.0",
+    "eslint-import-resolver-typescript": "^3.5.2",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.1.6",
+    "eslint-plugin-jest-dom": "^4.0.3",
+    "eslint-plugin-n": "^15.6.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.1.1",
```

Comment:
Same here, do we use this?

Reply:
I believe so, yes. Removed the other ones though.